src/OrchardCore.Modules/OrchardCore.Contents/Views/Stereotype.SaveDraftButton.cshtml

```diff
@@ -0,0 +1,5 @@
+@{
+    Model.Metadata.Type = "Content_SaveDraftButton";
```

Comment:
You don’t need create this file, it’s not related to stereotype

Reply:
done.